mautic/mautic
# Zapier plugin

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Automated tests included? | Y
| Related user documentation PR URL | https://mautic.org/docs/en/plugins/zapier.html
| Related developer documentation PR URL | https://github.com/mautic/mautic-zapier, https://github.com/mautic/developer-documentation/pull/98
| Issues addressed (#s or URLs) | /
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This plugin adds very simple Zapier plugin which will just shows a list of preconfigured Zap templates, adds a link to docs and mention the API must be enabled.

To do this easily I added "Secondary Plugin Description". Until now the description text showed up only in the H3 showing in 8/12 of the popup. It wasn't convenient to show the Zap templates there. So I implemented simple solution. If the description is separated by `\n---\n` (`\n` means new line) then the part after this separator will show up below the logo and H3 in the full width. The separator was taken from the YAML syntax where it "indicates the start of a document".

#### Steps to test this PR:
1. Go to plugins.
2. Hit install.
3. The Zapier plugin will show up.
4. Open it.
5. This is what you should see:

![screen shot 2018-03-09 at 12 10 43](https://user-images.githubusercontent.com/1235442/37205238-578caeda-2394-11e8-9912-b08fa8ed3926.png)

It will have more than one Zap template in the near future.
